pdns/pdnsutil.cc

```diff
@@ -1905,7 +1916,7 @@ try
     cout<<"                                   List all zone names"<<endl;;
     cout<<"list-tsig-keys                     List all TSIG keys"<<endl;
     cout<<"rectify-zone ZONE [ZONE ..]        Fix up DNSSEC fields (order, auth)"<<endl;
-    cout<<"rectify-all-zones                  Rectify all zones."<<endl;
+    cout<<"rectify-all-zones [quiet]          Rectify all zones. Optionnally quiet output with errors only"<<endl;
```

Comment:
if you could replace 'optionnally' with 'optionally' I'd be grateful :)

Reply:
Sure... My english is so bad :/

Reply:
I'm sure we could hook up aspell in a pre-commit hook or something ;)

Reply:
> I'm sure we could hook up aspell in a pre-commit hook or something ;)


or in travis :laughing:

Reply:
or in vim like:
set spell spelllang=en,ru,de

and put dictionaries from http://ftp.vim.org/vim/runtime/spell/ into ~/.vim/spell :-)

Reply:
But who still uses vim ?

I mean, it's 2018...